src/libcore/iter.rs

```diff
@@ -2553,10 +2553,7 @@ pub trait Extend<A> {
     ///
     /// # Examples
     ///
-    /// Basic usage:
```

Comment:
As I go through the docs, I'm adding them, so this will become gradually so over time.